projects/core/src/error-handling/effects-error-handler/cx-error-handler.effect.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { Actions, createEffect } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { filter, tap } from 'rxjs/operators';
+import { FeatureConfigService } from '../../features-config';
+import { ErrorAction } from '../../model/index';
+import { EffectsErrorHandlerService } from './effects-error-handler.service';
+
+@Injectable()
+export class CxErrorHandlerEffect {
```

Comment:
[MINOR]
Let's add JSDocs for 2 classes - for this class and for `EffectsErrorHandlerService`.